.github/workflows/developer_onboarding_notification.yml

```
@@ -0,0 +1,73 @@
+
+name: PR Merge Comment
```

I'm thinking of a more descriptive name, what do you think @Mayank77maruti? 
```suggestion
name: Celebrating Initial Contributions
```

---

Yup a more descriptive name would be better.